content/blog/posts/2022-09-30-conference-basics.md

```diff
@@ -0,0 +1,67 @@
+---
+authorId: hannakim
+categories:
+  - devlife
+  - developertips
+  - community
+date: '2022-08-30'
+slug: conference-101
+title: '5 Takeaways From Attending My First Tech Conference'
+---
+
+## In-person Conferences Are Back!
+
+I had the chance to attend my first in-person tech conference, [CascadiaJS](https://2022.cascadiajs.com/), earlier this month and I'm very glad to have live events back in full swing again!
+
+As software engineers, we're lucky to be in a field that allowed a fairly easy migration to remote work. Virtual conferences and hackathons were plenty available the last few years, but there is something special about in-person events. They litate connection in a way I don't think we can quite replicate digitally, though as metaverse creation tools keep evolving, this may change.
```

Comment:
What does `litate` mean? Was this intentionally changed from `facilitate`?

Reply:
It means forked, like leaves on a plant ...but typo from me 😂